src/main/java/calculator/domain/Numbers.java

```diff
@@ -0,0 +1,59 @@
+package calculator.domain;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class Numbers {
+
+    private static final String REGX_DELIMITER = ",|:";
+    private static final String REGX_CUSTOM_DELIMITER = "//(.)\n(.*)";
+    private static final Pattern PATTERN_REGX_CUSTOM = Pattern.compile(REGX_CUSTOM_DELIMITER);
+    private static final int POSITION_CUSTOM_DELIMITER = 1;
+    private static final int POSITION_OTHER_EXPRESSION = 2;
+    private static final String COMMA = ",";
+    private static final List<Number> ZERO_NUMBERS = Collections.singletonList(new Number(0));
+    private static final Number ZERO = new Number(0);
```

Comment:
아래와 같이 변경할 수 있을 것 같아요. 😃 
```suggestion
    private static final Number ZERO = new Number(0);
    private static final List<Number> ZERO_NUMBERS = Collections.singletonList(ZERO);
```